cabby/model/landmark_recognition/pivot_recognition.py

```diff
@@ -48,6 +48,9 @@
 flags.DEFINE_string("model_path", None,
           "The path to save the model.")
 
+flags.DEFINE_string("pivot_name", None,
+          "Pivot name: 'all'|main_pivot|near_pivot|end_pivot|beyond_pivot.")

```

Comment:
I used the already defined list of landmarks in walk.py and changed the flag to enum

Reply:
Hmm...  it doesn't look like your enum is what I was thinking it would be. I was thinking it would be something like:
```
class PivotType(Enum):
   ALL = 1
   MAIN_PIVOT = 2
   NEAR_PIVOT = 3
   END_PIVOT = 4
   BEYOND_PIVOT = 5
```

Then you can refer to these as PivotType.ALL, PivotType.MAIN_PIVOT, etc, and know you are getting consistent reference (and not having misspellings, etc).

If you use this, then you also don't need the EXTRACT_ALL_PIVOTS = "all" thing.

Reply:
Two things:
(1) I don't want to define in two places the pivot types. The pivots are already defined in walk.py except for ALL option.
(2) Even if I define:
landmark_dict = {k: v for v, k in enumerate(landmark_list)}
landmarks_types = enum.Enum('Landmarks', landmark_dict)
which is based on walk.py

flags.DEFINE_enum only except a list of strings and it makes landmarks_types  based on enum class redundant.

Then you can refer to these as PivotType.ALL, PivotType.MAIN_PIVOT, etc, and know you are getting consistent references (and not having misspellings, etc). -- I don't have a specific reference in the code, I just iterate over all options (except for EXTRACT_ALL_PIVOTS, which is defined once)

Reply:
Okay, sounds good -- let's leave as is then.